playbooks/system/update.yml

```diff
@@ -0,0 +1,31 @@
+---
+- hosts:
+    - master
+    - node
+    - build
+  remote_user: '{{ k8s_ssh_user }}'
+  become: true
+
+  tasks:
+  - name: Update CentOS packages
+    yum:
+      name: '*'
+      state: latest
+    notify:
+      - reboot vps
+      - wait to come up
+ 
+  handlers:
+
+    - name: reboot vps
+      command: /sbin/shutdown -r +1
```

Comment:
why do you use +1 ???
shutdown -r now

from manual:
"now" is an alias for "+0", i.e. for triggering an immediate shutdown.
If no time argument is specified, "+1" is implied.